cmd/soroban-rpc/internal/daemon/daemon.go

```diff
@@ -133,6 +133,7 @@ func newCaptiveCore(cfg *config.Config, logger *supportlog.Entry) (*ledgerbacken
 func MustNew(cfg *config.Config) *Daemon {
 	logger := supportlog.New()
 	logger.SetLevel(cfg.LogLevel)
+	supportlog.SetLevel(cfg.LogLevel)
```

Comment:
why is this change included?

Reply:
Because otherwise when you do `supportlog.Debugf(...)` you get no logs :/ I get that we should probably have component-specific loggers, but since we don't, why do we not just use the global logger everywhere, anyway?